alignak_module_backend/broker/module.py

```diff
@@ -247,9 +247,19 @@ def update(self, data, obj_type):
                     'ls_long_output': data['long_output'],
                     'ls_perf_data': data['perf_data'],
                     'ls_acknowledged': data['problem_has_been_acknowledged'],
+                    'ls_acknowledgement_type': data['acknowledgement_type'],
                     'ls_downtimed': data['in_scheduled_downtime'],
                     'ls_execution_time': data['execution_time'],
-                    'ls_latency': data['latency']
+                    'ls_latency': data['latency'],
+
+                    'ls_passive_check': data['check_type'] == '1',
```

Comment:
Not sure, but if I remember, `data['check_type']` is an int, so need test with `1`, not `'1'`